src/components/Heading.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+type HeadingProps = {
+  title: string;
+  subtitle?: string;
+  center?: boolean;
+};
+
+export function Heading({ title, center, subtitle }: HeadingProps) {
+  return (
+    <div className={center ? 'text-center' : 'text-start'}>
+      <h2 className="text-2xl font-bold dark:text-white">{title}</h2>
+      <h4 className="mt-2 font-light text-neutral-500 dark:text-neutral-400">
+        {subtitle}
+      </h4>
+    </div>
+  );
+}
+
+Heading.defaultProps = {
+  center: false,
+  subtitle: '',
+};
```

Comment:
Because this component is going to be re-used in other places.

Example. I am using this component for both the Modals, (and modals themselves might and should have a variety), and the product card, and even the cart card you are working on, each relying on that set of heading styles. No need to copy paste styles when they can be extracted into a component.

It also helps in maintaining harmony between designs.